.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v**'
```

Comment:
In my projects I tend to tag with a `v` prefix, which I'm aware differs from the history of this repo. If there's a preference to stick with purely numeric tags we can change this.